packages/teleport/src/Nodes/useNodes.ts

```diff
@@ -191,15 +191,14 @@ export default function useNodes(ctx: Ctx, stickyCluster: StickyCluster) {
   };
 }
 
-function makeOptions(
-  clusterId: string,
-  serverId = '',
-  logins = [] as string[]
-) {
+function makeOptions(clusterId: string, node: Node | undefined) {
+  const nodeLogins = node?.sshLogins || [];
+  const logins = sortLogins(nodeLogins);
+
   return logins.map(login => {
     const url = cfg.getSshConnectRoute({
       clusterId,
-      serverId,
+      serverId: node?.id || '',
```

Comment:
I don't think so, but I added it since the node ID was being checked before.

I think we can remove it since this code only runs once the user clicked on Connect, and they can't do it unless there is a node to click in the first place.